.travis.yml

```diff
@@ -1,39 +1,38 @@
 ---
 language: node_js
 node_js:
-  - "4"
+  - "8"
 
 sudo: false
 
+dist: trusty
+
 cache:
   directories:
     - $HOME/.npm
     - $HOME/.cache # includes bowers cache
 
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-lts-2.4
-  - EMBER_TRY_SCENARIO=ember-lts-2.8
-  - EMBER_TRY_SCENARIO=ember-release
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
+  - EMBER_TRY_SCENARIO=ember-3.4.0
```

Comment:
Can this be 3.4 instead of 3.4.0 to get the latest as opposed to the first version of 3.4?

Reply:
Will try, in next PR if that's fine. I wonder if it's not taking the latest version automatically as i took the code from liquid-fire repo but maybe not looking at https://github.com/britishgas-engineering/ember-appointment-slots-pickers/pull/7/files#diff-d20f4baf61c4397c467d6218d0ba6a04R9. I'll check

Reply:
(Added to the bullet list in https://github.com/britishgas-engineering/ember-appointment-slots-pickers/pull/7#issue-340477983)